contrib/golang/upstreams/http/tcp/test/test_data/http2dubbo-example/filter.go

```diff
@@ -0,0 +1,277 @@
+package main
```

Comment:
@wbpcode Could you please take a quick look at this user-side example? It allows people conver http request with any user-defined protocol(i.e. json) into any other TCP based request (i.e. kinds of RPC, in this example it's dubbo). 
We want to make sure it's a proper design.

Then we'll take care the implementation details, thanks.

Reply:
I have update the annotation for http2dubbo-example, please refresh the newest code, thanks~